src/main/java/uk/nhs/digital/ps/migrator/model/hippo/NationalIndicatorMigrator.java

```diff
@@ -5,8 +5,10 @@
 import java.util.Iterator;
 import java.util.List;
 
-import org.apache.poi.ss.usermodel.Row;
+import javax.sql.DataSource;
 
+import org.apache.poi.ss.usermodel.Row;
+import org.slf4j.LoggerFactory;
```

Comment:
While removing the DataSource import, would you mind cutting this one out too, Jon?